dsf-bpe/dsf-bpe-process-feasibility/src/main/java/org/highmed/dsf/bpe/service/ModifyQueries.java

```diff
@@ -15,6 +20,33 @@ public ModifyQueries(FhirWebserviceClientProvider clientProvider, TaskHelper tas
 	@Override
 	protected void doExecute(DelegateExecution execution) throws Exception
 	{
-		// TODO: implement
+		Boolean needsConsentCheck = (Boolean) execution.getVariable(Constants.VARIABLE_NEEDS_CONSENT_CHECK);
+		Boolean needsRecordLinkage = (Boolean) execution.getVariable(Constants.VARIABLE_NEEDS_RECORD_LINKAGE);
+		boolean idQuery = Boolean.TRUE.equals(needsConsentCheck) || Boolean.TRUE.equals(needsRecordLinkage);
```

Comment:
Why not just use `needsConsentCheck || needsRecordLinkage` without Boolean.True.equals